enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -771,7 +771,33 @@ To answer this question, we need to do the following:
 * Verify that OpenShift Service Mesh will include support for this feature in time for this EP.
 * Evaluate any potential security concerns around this feature.
 
-**Resolution**: TBD.
+**Resolution**: The answer depends on what version of OSSM is selected for dev
+preview. If we use OSSM 2.3, we will **NOT** enable ReferenceGrants. If we use OSSM 2.4,
+we will enable ReferenceGrants. This is because ReferenceGrants are non-functional in
+OSSM 2.3, but functional in OSSM 2.4.
+
+OSSM 2.3 uses Istio 1.14 and Istio 1.14 doesn't support ReferenceGrants; therefore,
+in OSSM 2.3, ReferenceGrants are **non-functional**. This means that, by default,
+Gateway API objects in OSSM 2.3 can reference objects across namespace boundaries,
+such as an HTTPRoute referencing a service in another namespace. A ReferenceGrant
+CRD has no impact on this functionality.
+
+OSSM 2.4 uses Istio 1.16 and Istio 1.16 supports ReferenceGrants; therefore,
+in OSSM 2.4, ReferenceGrants are **functional**. This means that, by default,
+Gateway API objects in OSSM 2.4 can **NOT** reference objects across namespace
```

Comment:
Minor grammar correction:
```suggestion
Gateway API objects in OSSM 2.4 **CANNOT** reference objects across namespace
```
Using both boldface **and** capitals seems a bit much though.  * grin *.

Reply:
done